assessments/guide/security-reviewer.md

```diff
@@ -43,4 +43,35 @@ expected to conduct an in-depth review described in the [Security Assessment
 Guide](./). Security Reviewers should seek the approval of the other
 participating Security Reviewers and at least 1 co-chair before merging.
 
+## Conflict of interest
 
+There is a possibility of conflict of interest that can arise between a security reviewer and
+the project being reviewed due to the closely-knit nature of the community. Having clear
+guidelines for conflict of interest situations are important to prevent:
+
+- Individuals from intentionally or unintentionally promote their own company's project
+- SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
+- Security reviews being stalled while groups discuss whether this person or that person should be allowed to participate
+
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
+Hard conflicts makes a reviewer ineligible to review a project.
+A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+A reviewer with a soft conflict is required to document the nature of the conflict in the
+security assessment documentation. It is the responsibility of the security reviewer
+to make known his/her own conflict of interests.
+
+For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts.
```

Comment:
As much as possible we try to avoid conflict, but in cases where we need to go with a conflict, we should document the justification for allowing the conflict. This could be as specific as "This reviewer is the expert in this field and therefore we require their input" or as conventional as "No one else has the bandwidth to take this on at this time so we are letting an exception" -- over time, if we are documenting and reviewing the reason for exception it will allow us to keep the process honest.